vm/runtime/src/lib.rs

```diff
@@ -182,9 +183,25 @@ pub trait Syscalls {
         Ok(data_commitment_v1_to_cid(&comm_d))
     }
     /// Verifies a sector seal proof.
-    fn verify_seal(&self, _vi: &SealVerifyInfo) -> Result<(), Box<dyn StdError>> {
-        // TODO
-        todo!()
+    fn verify_seal(&self, vi: &SealVerifyInfo) -> Result<(), Box<dyn StdError>> {
+        let commd = cid_to_data_commitment_v1(&vi.unsealed_cid)?;
+        let commr = cid_to_replica_commitment_v1(&vi.on_chain.sealed_cid)?;
+        let miner_addr = Address::new_id(vi.sector_id.miner);
+        let prover_id = <[u8; 32]>::try_from(miner_addr.payload_bytes().as_slice())?;
```

Comment:
This will error always (unless the payload slice is length 32 lol) should initialize an empty `Commitment` (would be nice to use the type from the api for readability) and then copy from the payload bytes

Reply:
We are sure it isn't always 32 lol. It's an ID address payload, its dynamic size and generally very small

Reply:
Oops, this got posted after Dustin made his changes. copy_from_slice will fail still since its not 32 bytes.